pom.xml

```diff
@@ -371,7 +371,7 @@ under the License.
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-changelog-plugin</artifactId>
-        <version>2.2</version>
+        <version>3.0.0-SNAPSHOT</version>
```

Comment:
Not sure if we're ready to change the version or not

Reply:
I've changed it back to 2.2